COPYING.md

```diff
@@ -7,38 +7,34 @@ AliceVision is licensed under the [MPL2 license](LICENSE-MPL2.md).
 modules are based on the libmv code released under MIT, see [LICENSE-MIT-libmv.md](LICENSE-MIT-libmv.md).
```

Comment:
If you cite there libmv there it should be fair to cite OpenMVG too.

Reply:
The intention was to explain the license change from MIT to MPL for the modifications.
But I added a sentence for openMVG (and for cmpmvs to be coherent).